Dockerfile

```diff
@@ -35,14 +41,20 @@ COPY pkg/ pkg/
 COPY util/ util/
 
 # Build
+ARG package=.
```

Comment:
I don't see this `package` ARG been used/passed anywhere!

Reply:
Fixed.